### PR TITLE
Clarify behavior of `placement_group_capture_child_tasks` in docs

### PR DESCRIPTION
**Pull request by amogkam**
Explicitly specify that `placement_group_capture_child_tasks` value needs to be set for every actor and the value is not inherited from its parent actor.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/ray-core/scheduling/placement-group.rst [hunk 1]
```diff
@@ -577,6 +577,11 @@ child tasks and actors to the same placement group, specify ``PlacementGroupSche
   :start-after: __child_capture_disable_pg_start__
   :end-before: __child_capture_disable_pg_end__
 
+.. warning::
+  The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If are creating nested actors of depth greater than 1
```
Comment (amogkam) [inline doc/source/ray-core/scheduling/placement-group.rst:581]:
```suggestion
  The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If you are creating nested actors of depth greater than 1 
```

**Review by rkooo567 — APPROVED** [review]

**Comment by rkooo567** [comment]
cc @angelinalg

**Comment by rkooo567** [comment]
doc build failure @amogkam

**Comment by can-anyscale** [comment]
please rebase, the readsthedoc build is fixed on lastest master, thankks

File: doc/source/ray-core/scheduling/placement-group.rst [hunk 2]
```diff
@@ -577,6 +577,11 @@ child tasks and actors to the same placement group, specify ``PlacementGroupSche
   :start-after: __child_capture_disable_pg_start__
   :end-before: __child_capture_disable_pg_end__
 
+.. warning::
+  The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If you are creating nested actors of depth greater than 1
```
Comment (angelinalg) [inline doc/source/ray-core/scheduling/placement-group.rst:581]:
```suggestion
  The value of ``placement_group_capture_child_tasks`` for a given actor isn't inherited from its parent. If you're creating nested actors of depth greater than 1 
```

File: doc/source/ray-core/scheduling/placement-group.rst [hunk 3]
```diff
@@ -577,6 +577,11 @@ child tasks and actors to the same placement group, specify ``PlacementGroupSche
   :start-after: __child_capture_disable_pg_start__
   :end-before: __child_capture_disable_pg_end__
 
+.. warning::
+  The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If you are creating nested actors of depth greater than 1 
+  that should all use the same placement group, ``placement_group_capture_child_tasks`` should be explicitly set for each actor.
```
Comment (angelinalg) [inline doc/source/ray-core/scheduling/placement-group.rst:582]:
```suggestion
  and should all use the same placement group, you should explicitly set ``placement_group_capture_child_tasks`` explicitly set for each actor.
```

**Review by angelinalg — APPROVED** [review]
just some nits

**Pull request merged**